src/migrate/migrate.ts

```diff
@@ -23,9 +21,9 @@ export const migrate = async (
   message: Message,
   chunkSize: number
 ): Promise<boolean> => {
-  const { migrationId = 0, channelId } = message;
+  const { migrationId = 0 } = message;
```

Comment:
If the migrationId defaults to zero then it means something is not right hence the migration should not continue from this point.

Reply:
made the changes chief.